Admin/AbstractAdminExtension.php

```diff
@@ -186,4 +186,16 @@ public function configureActionButtons(AdminInterface $admin, $list, $action, $o
     {
         return $list;
     }
+
+//    TODO: replace this doc block to inheritdoc in NEXT_MAJOR
+//    /**
+//     * Setting to true will enable mosaic button for the admin screen.
+//     * Setting to false will hide mosaic button for the admin screen.
+//     *
+//     * @param AdminInterface $admin
+//     * @param bool $isShown
+//     */
```

Comment:
That's only for interface methods, which are commented too.

Please uncomment the PHPDoc and leave the TODO only.